api/document/js/schema.ts

```diff
@@ -39,7 +39,7 @@ const schema = new Schema({
       toDOM: node => [`h${node.attrs.depth}`, { class: 'node-heading' }, 0],
     },
     para: {
-      content: 'inline? block*',
+      content: 'inline block*',
```

Comment:
So it seems like this is the actual fix... interesting... so was the `inline` basically being deleted, leaving only the `block`, e.g. a paragraph with no `<content>` but still having children?

Reply:
You got it, the `para`'s schema was allowing empty `content`, so deleting that text wasn't actually deleting the `para`. Combine that with an assumption of text with loading `para`s and we kill two birds with one stone by making the schema more rigid.

We *may* want to allow `(inline block*) | block+` in the future, I'm not sure. I suspect we'd need to add custom logic to the deletion commands to "fix" broken docs.